sklearn/cross_decomposition/_pls.py

```diff
@@ -65,6 +65,10 @@ def _get_first_singular_vectors_power_method(X, Y, mode="A", max_iter=500,
         if norm_y_weights:
             y_weights /= np.sqrt(np.dot(y_weights, y_weights)) + eps
 
+        # Replace weights that are all close to zero with zeros
```

Comment:
The + eps in the line below is already supposed to help with stability. Should we just increase it?